app/mailers/grant_creator_request_review_mailer.rb

```diff
@@ -1,15 +1,21 @@
 class GrantCreatorRequestReviewMailer < ApplicationMailer
   def approved(request:)
-    @request   = request
-    @requester = @request.requester
+    set_attributes(request: request)
 
     mail(to: @requester.email, subject: I18n.t('mailers.grant_creator_request_review.approved.subject'))
```

Comment:
See `config/locales/mailers/` directory for the subject lines of these emails.  The application name can be passed as an argument to the translation.